### DISCUSSION TITLE
Livewire website login with github showing wrong avatar

### DISCUSSION THREAD

**Discussion by Cyapow** [livewire/livewire#7953] [4. General]
Seems odd but had noway to report this on the site when logging into [https://livewire.laravel.com/](https://livewire.laravel.com/) with github it shows the wrong avatar for me

![SCR-20240215-hvx](https://github.com/livewire/livewire/assets/5711362/1c6ff69e-ed81-4f9e-8af9-687658977dcf)

**Comment by joshhanley** [comment]
@Cyapow is it using an old avatar? I suspect it just caches it the first time you login.
